.goreleaser.yml

```diff
@@ -20,3 +18,23 @@ checksum:
 
 archives:
   - name_template: '{{ .ProjectName }}-v{{ .Version }}-{{ .Os }}-{{ .Arch }}'
+
+brews:
+  - name: subo
+    description: Suborbital CLI for building and deploying WebAssembly-based projects
+    homepage: https://suborbital.dev
+    folder: Formula
+
+    tap:
+      owner: suborbital
+      name: homebrew-subo
+      token: '{{ .Env.HOMEBREW_TAP_GITHUB_TOKEN }}'
+
+    commit_author:
+      name: suborbital
+      email: 12345678+suborbital[bot]@users.noreply.github.com
```

Comment:
Need to confirm this number before merge.

Reply:
Where does this number come from?

Reply:
I believe its randomly generated and I'll need to get the bot to commit something first to find out what it is.

Reply:
It stays but it can break. It's kind of a weird hack to use a bot as the committer. We could use a static account too. Do you have any other ideas for who should be doing the writing?

Reply:
we could create an automation@suborbital.dev account? hm... I'm not sure actually

Reply:
I actually changed this to be the original committers name and email. That means that the author of the tag should get the credit on that PR. They will also get a push notification if it were to fail in CI. I think this is best.

What do you think?

Reply:
Nice! Didn't know we could do that :)